README.md

```diff
@@ -1,31 +1,348 @@
 # Hyperion
```

Comment:
Maybe have an initial section describing our 'flavor' of REST. Without going into detail, state how we handle API versioning, response types (json vs xml etc), how we handle error codes (just round down to nearest 100 with the specifics of the code residing in the structured error in the response).  I realize you speak to each of these below, but it helps to have a lay of the land before getting dragged into detail.